.github/scripts/generate_ci_workflows.py

```diff
@@ -54,7 +54,11 @@ def PyTorchLinuxWorkflow(
     test_runner_type: str,
     on_pull_request: bool = False,
     enable_doc_jobs: bool = False,
+    enable_jit_legacy_test: YamlShellBool = "''",
```

Comment:
What is the advantage of using YamlShellBool with '' and 1s instead of just having Trues/Falses with bools? Is it for better compat between shell and script?

Reply:
> Is it for better compat between shell and script?

Yeah exactly; this variable gets set as an environment variable, so it's easier for both shell and Python scripts to consume it if false is represented as the empty string.